wcfsetup/install/files/lib/data/option/OptionAction.class.php

```diff
@@ -175,7 +175,7 @@ protected function fetchRewriteRules() {
 				$content = <<<SNIPPET
 RewriteCond %{SCRIPT_FILENAME} !-d
 RewriteCond %{SCRIPT_FILENAME} !-f
-RewriteRule ^/{$path}(.*)$ {$path}/index.php?$1 [L,QSA]
+RewriteRule ^{$path}/(.*)$ {$path}/index.php?$1 [L,QSA]
```

Comment:
Are you positive that the leading slash must be removed?

Reply:
Yes; I fixed that in my own generator before but I forgot to commit it over here.
I'm just wondering if there might be a leading slash before the redirection-URL, but it seems to work without it.
You might want to wait for an answer to the first linked thread.

see also:
https://community.woltlab.com/thread/280528-rewrite-regeln-werden-nicht-korrekt-generiert/
https://www.woltlab.com/article/24-konfiguration-von-benutzerfreundlichen-urls-seo-urls/#deefc738-apache-litespeed

Reply:
I have to add a little workaround for the root directory too, I'm going to push a new commit.